Assets/Scripts/PlayerScripts/HealLife.cs

```diff
@@ -0,0 +1,32 @@
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class HealLife : MonoBehaviour
+{
+    private HealthController healthController = default;
+    private PlayerController _playerController = default;
+    private InputAction _inputHeal = default;
+    void Start()
```

Comment:
add space here i will not review you if you ever do this again >:(

Reply:
ya viejo loco